### PR TITLE
Feature/baseline

### PR DESCRIPTION
**Pull request by RCRobotics**
Got a basic version of the robot code working.

### CODE REVIEW

File: source/RoboticsClub/Robot/robot_control-fall_2017.c [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+int lastBtn6dState = 0;
+
+void updateLaunch()
+{
+    int currentBtnState = vexRT[Btn6D];
+
+    // Only launch when last state was not pressed and current state is pressed
+    if(currentBtnState != lastBtn6dState && currentBtnState)
+    {
+        motor[launchServo] = launchToggle ? 100 : -100;
```
Comment (TyGriffin3) [inline source/RoboticsClub/Robot/robot_control-fall_2017.c:144]: Should be 128: -128

**Review by TyGriffin3 — CHANGES_REQUESTED** [review]
Launch toggle 128: -128

File: source/RoboticsClub/Robot/robot_control-fall_2017.c (hunk 1 above)
Comment (TyGriffin3) [reply source/RoboticsClub/Robot/robot_control-fall_2017.c]: The range here is, indeed, 128:-128. However, we don't need the full power of the motor here, we just need to be able to move a lever enough to release the catapult.

Comment (RCRobotics) [reply source/RoboticsClub/Robot/robot_control-fall_2017.c:144]: The range here is, indeed, 128:-128. However, we don't need the full power of the motor here, we just need to be able to move a lever enough to release the catapult.

**Pull request merged**